spec/index.bs

```diff
@@ -82,9 +82,39 @@ Issue: figure out if we need this section here. CSP has none, SRI has one and it
     libraries that perform auxiliary tasks (e.g. analytics, performance
     monitoring). To ensure that none of these components introduces DOM
     XSS vulnerabilities, author defines a Trusted Type policy in the
-    templating policy and enables the enforcement for the DOM sinks.
-*   Issue: Something about reducing the review surface
-
+    templating policy and enables enforcement for the DOM sinks.
+*   A large team maintains a complex client-side application.
+    They create a number of Trusted Types policies so that
```

Comment:
I think you could remove the details of how the policies behave, and focus that there is a few of them and that they are reviewed. The current description might be too detailed for this section I feel.